docs/service.md

```diff
@@ -0,0 +1,58 @@
+---
+id: service
+title: Register as System Service
+---
+
+## Systemd Service File
+
+If you run Gotify from the executable, you can easily register it as a system service.
+On Debian based systems make the file
+
+```bash
+/etc/systemd/system/gotify.service
+```
+with the following content:
+
+```bash
+[Unit]
+Description=Gotify Server
+After=network.target
+
+[Service]
+Type=simple
+User=gotify
+ExecStart=/path/to/gotify-executable
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Now register the service with
+
+```bash
+$ systemctl daemon-reload
+```
+
+Now you should be able to start and stop the server with
+
+```bash
+$ service gotify start
+$ service gotify stop
+```
+
+and it restarts automatically in case of failure
+
+## Explanation
+
+After=network.target ensures Gotify is started after the network services when rebooting. When using MySQL add another line
```

Comment:
`After=network.target` and other snippets from code could also be formatted as code.